public_html/friendship.html

```diff
@@ -166,7 +166,10 @@ <h4 class="panel-title">
 	
 <div class="row"> 
 <p></p></div>
-	
+<hr>
+<div class="row">
+<p><center>Have any questions? Contact the Fellowship Vice President at <a href="mailto:apo-fellowship@wpi.edu">apo-fellowship@wpi.edu</a>!
```

Comment:
Is it supposed to be apo-fellowship@wpi.edu or gr-apo-fellowship@wpi.edu? Same question for all the other email addresses below.

Reply:
fellowship: apo-fellowship@wpi.edu
service: apo-service@wpi.edu
membership: apo-membership@wpi.edu
history: apo-historian@wpi.edu 
leadership: gr-apo-leadership@wpi.edu